packages/avatar/contracts/nft-collection/ERC721BurnUpgradeable.sol

```diff
@@ -11,18 +11,8 @@ import {ERC721Upgradeable} from "@openzeppelin/contracts-upgradeable-5.0.2/token
  * @notice Baseline ERC721 contract to be used by the NFTCollection contract
  * @dev provides the "burn memory" functionality: keeping track of who burned what token
  */
-abstract contract ERC721BurnMemoryUpgradeable is ERC721Upgradeable {
+abstract contract ERC721BurnUpgradeable is ERC721Upgradeable {
     struct ERC721BurnMemoryUpgradeableStorage {
```

Comment:
Contract name was update but the struct name did not, lets align it.

Reply:
We decided to removed the base clase completly.